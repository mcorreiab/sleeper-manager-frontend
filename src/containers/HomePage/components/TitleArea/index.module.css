.title {
  font-size: 2.4375rem;
  font-weight: 400;
  margin-bottom: 1rem;
}

.title span {
  color: #f26506;
}

.description {
  font-size: 1.25rem;
  font-family: "Open Sans", sans-serif;
}

@media screen and (min-width: 768px) {
  .description {
    font-size: 1.4rem;
  }
}

@media screen and (min-width: 768px) {
  .title {
    font-size: 2.85rem;
  }
}
