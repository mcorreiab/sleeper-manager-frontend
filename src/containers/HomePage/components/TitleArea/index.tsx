import React, { FunctionComponent } from "react";
import styles from "./index.module.css";

interface Props {
  className: string;
}

const titleArea: FunctionComponent<Props> = ({ className }) => (
  <div className={className}>
    <h2 className={styles.title} data-testid="page-title">
      Tool to easy your life at <span>sleeper</span> fantasy platform
    </h2>
    <p className={styles.description}>
      Insert your username to check all players with non ok injury status
    </p>
  </div>
);

export default titleArea;
