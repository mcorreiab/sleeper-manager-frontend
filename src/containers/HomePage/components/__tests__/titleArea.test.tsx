import renderer from "react-test-renderer";
import TitleArea from "../titleArea";

it("should render the title area with no props", () => {
  const tree = renderer.create(<TitleArea />).toJSON();
  expect(tree).toMatchSnapshot();
});
