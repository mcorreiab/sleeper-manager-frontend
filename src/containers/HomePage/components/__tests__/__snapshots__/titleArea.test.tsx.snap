// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the title area with no props 1`] = `
<div>
  <h2
    className="text-[2.4375rem] font-normal mb-4"
  >
    Tool to easy your life at 
    <span
      className="text-[#f26506]"
    >
      sleeper
    </span>
     
    fantasy platform
  </h2>
  <p
    className="text-xl font-['Open Sans, sans-serif'] md:text-[1.4rem]"
  >
    Insert your username to check all players with non ok injury status
  </p>
</div>
`;
